lib/jormungandr/src/Cardano/Wallet/Jormungandr.hs

```diff
@@ -199,19 +204,25 @@ serveWallet
     -- ^ Callback to run before the main loop
     -> IO ExitCode
 serveWallet
-    Tracers{..} sTolerance databaseDir hostPref listen lj beforeMainLoop = do
-    installSignalHandlers (traceWith applicationTracer MsgSigTerm)
-    traceWith applicationTracer MsgStarting
-    traceWith applicationTracer $ MsgNetworkName $ networkDiscriminantVal @n
-    Server.withListeningSocket hostPref listen $ \case
-        Left e -> handleApiServerStartupError e
-        Right (wPort, socket) -> do
-            either id id <$> race
-                (daedalusIPC daedalusIPCTracer wPort $> ExitSuccess)
-                (serveApp socket)
+    Tracers{..}
+    sTolerance
+    databaseDir
+    hostPref
+    listen
+    backend
+    beforeMainLoop = do
```

Comment:
That's the kind of moment where I think it's totally fine to not be under 80-lines :grimacing: ...